bindings/py/cpp_src/bindings/engine/py_Region.cpp

```diff
@@ -50,7 +50,7 @@ namespace htm_ext
 		// This would be .py code calling getSpec() to get the Spec structure
 		// on an already instantiated class.
         py_Region.def("getSpec", &Region_t::getSpec);
-
+        py_Region.def("executeCommand", &Region_t::executeCommand);
```

Comment:
Note that this was the crucial thing. Exposing executeCommand to python.

Reply:
Oh, it looks like I missed that one when setting up the bindings.  Good catch.